### DISCUSSION TITLE
Type hints for the `js` module?

### DISCUSSION THREAD

**Discussion by Archmonger** [pyodide/pyodide#5254] [Q&A] [answered]
Is there an existing type hint stubs package for the JS module?

For example, to provide type hints for calls like this

```python
import js

js.document.title = "New window title"
```

**Comment by ryanking13** [comment, marked as answer]
There is a [webtypy](https://github.com/pyodide/webtypy) project, which is mostly maintained by @simonegiacomelli. Contributions are welcome.
